runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -71,10 +71,15 @@
 
 ////////////////////////////////////////
 //
-// This is the libfabric API version we need.
+// Libfabric API version
 //
 
-#define COMM_OFI_FI_VERSION FI_VERSION(1, 8)
+//
+// This is used to check that the libfabric version the runtime is
+// linked with in a user program is the same one it was compiled
+// against.
+//
+#define COMM_OFI_FI_VERSION FI_VERSION(FI_MAJOR_VERSION, FI_MINOR_VERSION)
```

Comment:
I'm not sure how much a concern this is, but if the `FI_*_VERSION` changes (either because we upgraded third-party libs or because the system version was updated) will we rebuild the runtime? I think not, but maybe we should?

Reply:
(Doesn't need to happen in this PR, just checking)

Reply:
If `CHPL_LIBFABRIC=libfabric` and the libfabric version changes because of a third-party upgrade then we'll rebuild the runtime, because the third-party installed files will necessarily be newer than the runtime object files in that case.  I'm not sure there's a dependable way to achieve the same thing for `=system`, though.